misc/dist/linux/godot.6

```diff
@@ -150,9 +150,6 @@ Disallow dumping the base types (used with \fB\-\-doctool\fR).
 \fB\-\-build\-solutions\fR
 Build the scripting solutions (e.g. for C# projects). Implies \-\-editor and requires a valid project to edit.
 .TP
-\fB\-\-gdnative\-generate\-json\-api\fR
-Generate JSON dump of the Godot API for GDNative bindings.
-.TP
```

Comment:
I guess `--dump-extension-api` should be added to the man and completion files, but they are probably quite outdated in general.